Steps4Impact/Journey/Cells/CurrentMilestoneCell.swift

```diff
@@ -0,0 +1,187 @@
+//
+//  MilestoneCell.swift
+//  Steps4Impact
+//
+//  Created by Aalim Mulji on 11/17/19.
+//  Copyright © 2019 AKDN. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+struct CurrentMilestoneContext : CellContext {
+  var identifier: String = CurrentMilestoneCell.identifier
+  var sequence = 0
+  var distance = 0
+  var name = ""
+  var flagName = ""
+  var journeyMap = ""
+  var description = ""
+  var title = ""
+  var subTitle = ""
+  var media = ""
+  var content = ""
+  var status : MilestoneStatus = .current
+  var progress : CGFloat = 0.0
+}
+
+class CurrentMilestoneCell: ConfigurableTableViewCell {
+
+  static let identifier: String = "CurrentMilestoneCell"
+
+  let circle : UIView = {
+    var view = UIView(frame: .zero)
+    view.translatesAutoresizingMaskIntoConstraints = false
+    view.layer.cornerRadius = 12
+    return view
+  }()
+
+  let verticalBar : UIView = {
+    var view = UIView(frame: .zero)
+    view.translatesAutoresizingMaskIntoConstraints = false
+    view.frame.size.width = 2
+    return view
+  }()
+
+  let verticalProgressBar : UIView = {
+    var view = UIView(frame: .zero)
+    view.translatesAutoresizingMaskIntoConstraints = false
+    view.frame.size.width = 2
+    return view
+  }()
+
+  let progressCircle : UIView = {
+    var view = UIView(frame: .zero)
+    view.translatesAutoresizingMaskIntoConstraints = false
+    view.layer.cornerRadius = 12
+    view.layer.applySketchShadow(color: Style.Colors.FoundationGreen, alpha: 0.5, x: 0, y: 0, blur: 8, spread: 0)
+    return view
+  }()
+
+  let milestoneCountLabel : UILabel = {
+    var label = UILabel()
+    label.translatesAutoresizingMaskIntoConstraints = false
+    label.font = UIFont.systemFont(ofSize: 14, weight: .regular)
```

Comment:
Can you use one of the types from the [StyleGuide](https://github.com/AgaKhanFoundation/WCF-iOS/blob/master/Steps4Impact/App/StyleGuide.swift#L61). You can use the helper `UILabel(typography: .smallRegular)` for this.